pkg/kubelet/apis/config/fuzzer/fuzzer.go

```diff
@@ -19,7 +19,7 @@ package fuzzer
 import (
 	"time"
 
-	"github.com/google/gofuzz"
+	fuzz "github.com/google/gofuzz"
```

Comment:
This seems like an odd change to include.

Reply:
probably my go imports did a mess, will revert it